test/1_api.js

```diff
@@ -119,6 +120,7 @@ describe('Server', function() {
 		storage.clear();
 		xhr = sinon.useFakeXMLHttpRequest();
 		clock = sinon.useFakeTimers();
+		server.createScript = function() {};
```

Comment:
the way I see it, flakiness -> indeterminism -> weirdness, so all these flakiness fixes are pretty weird

I can't wait to fix integration tests. It will be a freakshow for show.

sign...

:panda_face: